### ISSUE TITLE
At VA Champs, the tablets lagged behind the main FTC Live system

### ISSUE THREAD

**Issue by ftctechnh** [adwiii/nc-ftc-inspection-spark#11]
The scorekeepers were using 2.4GHz wireless Android tablets.  They could hear the audio cues to start the match, but their tablets would not display the updated scoring screen until 5 o 6 seconds later.

In other words, once the match started, it took a noticeable/perceivable amount of time before the tablet display got displayed.

**Comment by adwiii** [comment]
To clarify, the Score Trackers were stuck on the "Waiting for match start." for 5-6 seconds before the autonomous scoring page was loaded?

**Comment by ftctechnh** [comment]
Yes, that is our understanding.  They would see waiting for match to start before the 5 to 6 seconds it would show the scoring screen.

**Comment by ftctechnh** [comment]
HIGH Priority

**Comment by adwiii** [comment]
We have not been able to duplicate this in local testing. This leads us to suspect that it is wireless issue and hope that we can get a better feel for the behavior at the super regionals. The easiest solution is to hardwire the score tracker devies
